internal/jimm/juju/model_status_parser.go

```diff
@@ -20,6 +23,10 @@ import (
 	"github.com/canonical/jimm/v3/pkg/api/params"
 )
 
+var (
+	jqQueryDeadline = time.Second * 5
```

Comment:
please make this configurable (via charm config).. let jimm admin decide what is a sensible timout and we can default to something like 10, 20, 30 seconds..